docs/docs/index.md

```diff
@@ -134,6 +134,35 @@ with q.connection.pipeline() as pipe:
 
 `Queue.prepare_data` accepts all arguments that `Queue.parse_args` does.
 
+### Batch Enqueueing
+
+When using `enqueue_many`, the `batch` argument can be used to return a batch object instead of a list of jobs:
+
+```python
+batch = q.enqueue_many(
+  [
+    Queue.prepare_data(count_words_at_url, ('http://nvie.com',), job_id='my_job_id'),
+    Queue.prepare_data(count_words_at_url, ('http://nvie.com',), job_id='my_other_job_id'),
+  ],
+  batch=True,
+  batch_id='my_batch'
+)
+```
+
```

Comment:
Maybe this is a better API?
```python
batch = Batch.create(connection=redis)
batch.enqueue_many()
```

Reply:
This API raises a few questions, do we allow jobs from different queues to be put in the same batch?

Reply:
I think so. For example if a user wants to batch jobs enqueued to high, medium, and low queues, that seems ok to me.

Reply:
What if I adjust the APIs to have batching jobs be totally separate from enqueueing? Users would still enqueue jobs with Queue.enqueue_many, and then they would have the option to add the returned array of jobs to a Batch using a new `create` method, or add the jobs to an existing batch with the `add_jobs` method.

Reply:
> What if I adjust the APIs to have batching jobs be totally separate from enqueueing?

I don't think this is a good idea because this means the enqueueing and batching of jobs wouldn't be atomic. This may lead to hard to debug race conditions in the future if we were to put more functionalities into `Batch`.

> Users would still enqueue jobs with Queue.enqueue_many.

This is something we can still do, both of these should still be possible:
```python
queue.enqueue(foo, batch=batch_object)
queue.enqueue(foo, batch='batch_name')

queue.enqueue_many(job_1, job_2, batch=batch)
```